platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/creation/tipset/TipsetEventCreator.java

```diff
@@ -429,9 +428,7 @@ private GossipEvent assembleEventObject(@Nullable final EventDescriptor otherPar
                 selfId,
                 lastSelfEvent,
                 otherParent == null ? Collections.emptyList() : Collections.singletonList(otherParent),
-                nonAncientEventWindow.getAncientMode() == AncientMode.BIRTH_ROUND_THRESHOLD
-                        ? nonAncientEventWindow.pendingConsensusRound()
-                        : addressBook.getRound(),
+                ROUND_FIRST,
```

Comment:
Is the idea that you don't want anything being detained in time traveler jail yet? If so, an explanatory future work snippet could be helpful

Reply:
I actually can't recall why this change was made. Reverted it and tests still pass.